awscli/autocomplete/local/fetcher.py

```diff
@@ -25,6 +27,18 @@ def _get_argument(self, lineage, current_command, arg_name):
         if command:
             return command.arg_table.get(arg_name)
 
+    def get_operation_model(self, lineage, current_command, operation):
+        if current_command and len(lineage) > 1:
+            try:
+                service_name = self.SERVICE_MAPPER.get(lineage[1], lineage[1])
+                service_model = self._cli_driver.session.get_service_model(
+                    service_name)
+                operation = ''.join([part.capitalize()
+                                     for part in current_command.split('-')])
+                return service_model.operation_model(operation)
```

Comment:
There are other services that are renamed and therefore do not match what the client name is in botocore. I wonder instead if we can just grab the `service_model` or `operation_model` attached to the command object to make this handle renames more generically?